extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -13,48 +13,51 @@ public static function onParserFirstCallInit( Parser $parser ) {
 		global $wgEnableDiscussions;
 
 		if ( $wgEnableDiscussions ) {
-			$parser->setHook( self::TAG_NAME, [ 'EmbeddableDiscussionsController', 'render' ] );
+			$parser->setHook( static::TAG_NAME, [ 'EmbeddableDiscussionsController', 'render' ] );
```

Comment:
TYPE is defined like this:
const TAG_NAME = 'discussions';

I was using self::TYPE to access it, but changed it to static::TYPE based on the comment from SonarQube. I think in my context, self and static have the same effect. I see both styles used in app source code.